src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_variantContentPricing.html.twig

```diff
@@ -0,0 +1,29 @@
+{% import "@SyliusAdmin/Common/Macro/money.html.twig" as money %}
+
+<h4 class="ui top attached large header">{{ 'sylius.ui.pricing'|trans }}</h4>
```

Comment:
Is this file used anywhere? I cannot find usage. Also, won't it be possible to pass always a variant to this template, so we won't need to use two duplicated implementations in other files (one for simple product and another one for more separate variants)